.github/workflows/release.yml

```diff
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - 'v*'
+    branches:
+      - 'fix-release'
```

Comment:
Do you want to leave this here for future release debug?

Reply:
Whoops, no I meant to take that out. Good catch.